apps/backend/src/libs/modules/mapbox/libs/types/mapbox-construct-route-mapped-data.type.ts

```diff
@@ -0,0 +1,12 @@
+import { type LineStringGeometry } from "~/libs/types/types.js";
+
+type MapboxConstructRouteMappedData = {
```

Comment:
let's call it by the app's common conventions
```suggestion
type MapboxConstructRouteResponseDto = {
```

 And as for internal to this module type `MapboxConstructRouteResponseDto` - we can name it like `MapboxConstructRouteApiResponseDto`, but its name is relatively less important than this one, because it can be exported and used in the backend in the future